nx-X11/programs/Xserver/hw/nxagent/Keystroke.c

```diff
@@ -49,7 +49,7 @@ extern Bool nxagentIpaq;
 extern char *nxagentKeystrokeFile;
 
 #ifdef NX_DEBUG_INPUT
-int nxagentDebugInputDevices = 0;
+int nxagentDebugInputDevices = False;
```

Comment:
Commit "Keystroke.c: use Booleans where appropriate" is ok, but fix the commit message. In its last line it says: "o# Bitte geben Sie eine Commit-Beschreibung für Ihre Änderungen ein. Zeilen,"...